2-projects/1-todo-list/script.js

```diff
@@ -1,14 +1,59 @@
+let todos = [
+  { task: "Wash the dishes", completed: false },
+  { task: "Do the shopping", completed: false },
+];
+
+const ulEl = document.querySelector("ul");
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  for (let item of todos) {
+    // console.log(item.task);
+    let itemsList = item.task;
+
+    // create the element li
+    const liEl = document.createElement("li");
+    //class of the li
+    liEl.setAttribute(
+      "class",
+      "list-group-item d-flex justify-content-between align-items-center"
+    );
+    // amend the element to array
+    liEl.innerHTML = itemsList;
```

Comment:
Refer to line 11, the content of the **li** element shows the task so variable **itemsList** seems redundant.